File: Python/ProjectEuler_06.py
# Sum Square Difference -- Solved
# Solution = 25164150

from Timer import Timer
Stopwatch = Timer()

SumOfSquares = sum(n**2 for n in range(1,101))
SumSquared = sum(n for n in range(1,101))**2

Solution = SumSquared - SumOfSquares

print('The difference between the sum of the squares and the square of the sums is: ', Solution)
Stopwatch.stop()

File: MATLAB/ProjectEuler_06.m
%% Sum Square Difference -- Solved
clear, close, clc, disp(mfilename), tic;

Terms = 1:100;
SquaredTerms = [];

for i = 1:100
    SquaredTerms(i) = (Terms(i))^2;
end

SquaredSum = (sum(Terms))^2;
SumSquaredTerms = sum(SquaredTerms);
Solution = SquaredSum - SumSquaredTerms;

fprintf(['The difference between the sum of the squares and the square of ' ...
    'the sums is: %d\n'], Solution); toc;